### ISSUE TITLE
does not even start

### ISSUE THREAD

**Issue by emdete** [lukaslihotzki/pixelflood#9]
the app does not start on my android with logcat:

```
12-15 18:27:29.506 I/ActivityManager( 5015): START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=de.lihotzki.pixelflood/android.app.NativeActivity bnds=[843,404][1045,682]} from uid 10046
12-15 18:27:29.511 D/audio_route( 4747): Apply path: speaker-protected
12-15 18:27:29.522 D/audio_hw_primary( 4747): enable_snd_device: snd_device(95: vi-feedback)
12-15 18:27:29.522 D/audio_route( 4747): Apply path: vi-feedback
12-15 18:27:29.522 D/audio_hw_primary( 4747): enable_audio_route: usecase(24) apply and update mixer path: spkr-vi-record
12-15 18:27:29.522 D/audio_route( 4747): Apply path: spkr-vi-record
12-15 18:27:29.544 I/ActivityManager( 5015): Start proc 8086:de.lihotzki.pixelflood/u0a110 for activity de.lihotzki.pixelflood/android.app.NativeActivity
12-15 18:27:29.547 D/audio_hw_primary( 4747): enable_audio_route: usecase(1) apply and update mixer path: low-latency-playback
12-15 18:27:29.548 D/audio_route( 4747): Apply path: low-latency-playback
12-15 18:27:29.586 E/msm8974_platform( 4747): ramp_speaker_gain: Could not get ctl for mixer cmd - Left Speaker Gain or Right Speaker Gain, not applying speaker gain ramp
12-15 18:27:29.586 D/audio_hw_primary( 4747): out_write: retry previous failed cal level set
12-15 18:27:29.850 D/AndroidRuntime( 8086): >>>>>> START com.android.internal.os.RuntimeInit uid 10110 <<<<<<
12-15 18:27:30.005 D/AndroidRuntime( 8086): Calling main entry com.android.internal.os.ExecInit
12-15 18:27:30.005 D/Zygote  ( 8086): begin preload
12-15 18:27:30.014 D/vndksupport( 8086): Loading /vendor/lib64/hw/android.hardware.graphics.mapper@2.0-impl.so from current namespace instead of sphal namespace.
12-15 18:27:30.016 I/Zygote  ( 8086): Preloading shared libraries...
12-15 18:27:30.098 D/audio_hw_primary( 4747): disable_audio_route: usecase(1) reset and update mixer path: low-latency-playback
12-15 18:27:30.113 D/audio_hw_primary( 4747): disable_snd_device: snd_device(95: vi-feedback)
12-15 18:27:30.113 D/audio_hw_primary( 4747): disable_audio_route: usecase(24) reset and update mixer path: spkr-vi-record
12-15 18:27:30.128 D/Zygote  ( 8086): end preload
12-15 18:27:30.277 W/System  ( 8086): ClassLoader referenced unknown path: 
12-15 18:27:30.287 D/vndksupport( 8086): Loading /vendor/lib64/egl/libEGL_adreno.so from current namespace instead of sphal namespace.
12-15 18:27:30.298 D/libEGL  ( 8086): loaded /vendor/lib64/egl/libEGL_adreno.so
12-15 18:27:30.375 D/vndksupport( 8086): Loading /vendor/lib64/egl/libGLESv1_CM_adreno.so from current namespace instead of sphal namespace.
12-15 18:27:30.377 D/libEGL  ( 8086): loaded /vendor/lib64/egl/libGLESv1_CM_adreno.so
12-15 18:27:30.400 D/vndksupport( 8086): Loading /vendor/lib64/egl/libGLESv2_adreno.so from current namespace instead of sphal namespace.
12-15 18:27:30.401 D/libEGL  ( 8086): loaded /vendor/lib64/egl/libGLESv2_adreno.so
12-15 18:27:30.421 I/Adreno  ( 8086): QUALCOMM build                   : 8e59954, I0be83d0d26
12-15 18:27:30.421 I/Adreno  ( 8086): Build Date                       : 09/22/17
12-15 18:27:30.421 I/Adreno  ( 8086): OpenGL ES Shader Compiler Version: EV031.21.02.00
12-15 18:27:30.421 I/Adreno  ( 8086): Local Branch                     : O17A
12-15 18:27:30.421 I/Adreno  ( 8086): Remote Branch                    : 
12-15 18:27:30.421 I/Adreno  ( 8086): Remote Branch                    : 
12-15 18:27:30.421 I/Adreno  ( 8086): Reconstruct Branch               : 
12-15 18:27:30.423 D/vndksupport( 8086): Loading /vendor/lib64/hw/gralloc.msm8992.so from current namespace instead of sphal namespace.
12-15 18:27:30.428 I/Adreno  ( 8086): PFP: 0x00000000, ME: 0x00000000
12-15 18:27:30.432 W//system/bin/hwservicemanager(  362): getTransport: Cannot find entry android.hardware.configstore@1.0::ISurfaceFlingerConfigs/default in either framework or device manifest.
12-15 18:27:30.435 I/de.lihotzki.pixelflood( 8086): android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasWideColorDisplay retrieved: 0
12-15 18:27:30.441 I/ActivityManager( 5015): Displayed de.lihotzki.pixelflood/android.app.NativeActivity: +898ms
12-15 18:27:30.530 W/tzki.pixelflood( 8104): type=1400 audit(0.0:140230): avc: denied { read } for name="file-max" dev="proc" ino=857657 scontext=u:r:untrusted_app_25:s0:c512,c768 tcontext=u:object_r:proc:s0 tclass=file permissive=0
12-15 18:27:30.534 F/libc    ( 8086): Fatal signal 6 (SIGABRT), code -6 in tid 8104 (tzki.pixelflood), pid 8086 (tzki.pixelflood)
12-15 18:27:30.566 F/crash_dump64( 8109): crash_dump.cpp:298] failed to attach to thread 8104: Operation not permitted
12-15 18:27:30.566 I//system/bin/tombstoned( 3139): received crash request for pid 8086
12-15 18:27:30.569 F/libc    ( 8086): crash_dump helper failed to exec
12-15 18:27:30.569 W//system/bin/tombstoned( 3139): crash socket received short read of length 0 (expected 12)
12-15 18:27:30.599 I/ActivityManager( 5015): Process de.lihotzki.pixelflood (pid 8086) has died: fore TOP 
12-15 18:27:30.599 W/InputDispatcher( 5015): channel 'd638964 de.lihotzki.pixelflood/android.app.NativeActivity (server)' ~ Consumer closed input channel or an error occurred.  events=0x9
12-15 18:27:30.599 E/InputDispatcher( 5015): channel 'd638964 de.lihotzki.pixelflood/android.app.NativeActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
12-15 18:27:30.599 I/WindowManager( 5015): WIN DEATH: Window{d638964 u0 de.lihotzki.pixelflood/android.app.NativeActivity}
12-15 18:27:30.599 W/InputDispatcher( 5015): Attempted to unregister already unregistered input channel 'd638964 de.lihotzki.pixelflood/android.app.NativeActivity (server)'
12-15 18:27:30.599 W/zygote64( 5015): kill(-8086, 9) failed: No such process
12-15 18:27:30.599 W/ActivityManager( 5015): Force removing ActivityRecord{103b021 u0 de.lihotzki.pixelflood/android.app.NativeActivity t4738}: app died, no saved state
12-15 18:27:30.600 I/Zygote  ( 4746): Process 8086 exited due to signal (6)
12-15 18:27:30.632 W/ActivityManager( 5015): setHasOverlayUi called on unknown pid: 8086
12-15 18:27:30.641 W/zygote64( 5015): kill(-8086, 9) failed: No such process
12-15 18:27:30.641 I/zygote64( 5015): Successfully killed process cgroup uid 10110 pid 8086 in 42ms
12-15 18:27:31.202 W/OpenGLRenderer( 6132): Incorrectly called buildLayer on View: ShortcutAndWidgetContainer, destroying layer...
```

it's a copperhead android, may that is the cause?

**Comment by emdete** [comment]
i tested on another phone with lineageOS, android-version 7.1.0 which works fine. copperhead is android 8.1.0.

**Comment by lukaslihotzki** [comment]
This should be fixed in the latest version 1.0.3. Can you test it again?

**Comment by emdete** [comment]
sorry, my copperhead died recently so i can try on a different phone only - another  8.1.0-equipped one works fine.
